### ISSUE TITLE
Cartas híbridas

### ISSUE THREAD

**Issue by 0DASAC** [RodrigoToroIcarte/IIC2113-2023-1#191]
No me queda Claro como reconoser cuando una carta es hibrida. ya que en el enunciado solo mencionan cuales debemos implementar en esta entrega.

Por lo que entendi leyendo el enunciado general si una carta tiene mas de un tipo es Hibrida. Por lo tanto la siguente carta deberia ser hibrida:

  {
        "Title": "Shoulder Block",
        "Types": [
            "Maneuver",
            "Reversal"
        ],
        "Subtypes": [
            "Strike",
            "ReversalSpecial"
        ],
        "Fortitude": "0",
        "Damage": "3",
        "StunValue": "0",
        "CardEffect": "May only reverse a maneuver played after the card titled Irish Whip."
    }

**Comment by RodrigoToroIcarte** [comment]
Efectivamente. Una carta híbrida es una carta con más de un "_Type_". Pueden ser "Maneuver/Reversal", "Action/Maneuver" o "Action/Reversal".
